modules/ui/src/app/app-routing.module.ts

```diff
@@ -20,21 +20,33 @@ const routes: Routes = [
   {
     path: 'testrun',
     loadChildren: () =>
+<<<<<<< HEAD
+      import('./progress/progress.module').then(m => m.ProgressModule),
+=======
       import('./pages/testrun/progress.module').then(m => m.ProgressModule),
+>>>>>>> dev
```

Comment:
@sofyakurilova,
Please check how the conflicts are resolved.
We should re-work it.